apis/management.cattle.io/v3/backup_types.go

```diff
@@ -51,6 +51,8 @@ type EtcdBackup struct {
 type EtcdBackupSpec struct {
 	// cluster ID
 	ClusterID string `json:"clusterId,omitempty" norman:"required,type=reference[cluster]"`
+	// manual backup flag
+	Manual bool `yaml:"manual" json:"manual,omitempty"`
```

Comment:
do we need this field on an api level? if it used internally, you might need to drop it in schema.

Reply:
Example of dropping a field: https://github.com/rancher/types/pull/665/files#diff-d040f71d732be0b270d23ab28a54e726R623

Reply:
Actually, even if not used, this field can be useful for the info purpose.